makefile

```diff
@@ -177,6 +177,10 @@ post_install_libs:
 	cp src/lib/parse/*.h /usr/local/include/contextBroker/parse
 	cp $(CMAKE_BUILD_TYPE)/src/lib/parse/libparse.a  /usr/local/lib
 
+	cd /usr/local/include/contextBroker  && rm -rf mqtt && mkdir -p mqtt
```

Comment:
Maybe this first PR is a good point to include building documenation. I mean, to include in the https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/build_source.md document a bullet regarding libmqtt installation, just after libmicrohttp installation and in the same style (wget + make + etc.)

(Changes in the .spec to include mqtt dependencies would be another point... if you have proficiency in RPM .spec, please go ahead, if not don't worry, we can deal with it in the future).

Reply:
In addition to .spec, the doc/Dockerfile should be also adjusted...

Reply:
Fixed in 4896535

Mosquitto source build instructions are added to the build_source.md

The current list of dependencies cover what is needed by Mosquitto so I haven't changed the .spec.

Dockerfile is updated to handle Mosquitto installation